static/styles/zulip.css

```diff
@@ -1465,6 +1455,15 @@ td.pointer {
     opacity: 1;
 }
 
+.message_header + .selected_message {
+    /* Sticky message header overlaps with the box-shadow, so we add another box-shadow to
+       compensate for that. */
+    .messagebox-content {
+        box-shadow: inset 0 1px 0 2px hsl(215, 47%, 50%),
+            inset 0 0 0 2px hsl(215, 47%, 50%), 0 0 0 1px hsl(215, 47%, 50%);
```

Comment:
Should this HSL color be extracted as a variable?

And does this block need a dark theme analog?

Reply:
Extracting HSL color as variable doesn't make sense here since there is no prior of doing so here.
We don't need to do anything for the dark theme since the same color is used there too.